pages/1.11/release-notes/1.11.2/index.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.11.2
+title: Release Notes for 1.11.2
+menuWeight: 0
+excerpt: Release notes for DC/OS 1.11.2
+---
+
+These are the release notes for DC/OS 1.11.2.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.2/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+DC/OS 1.11.2 includes the following:
+
+Apache Mesos 1.5.x [change log](https://github.com/mesosphere/mesos/blob/27d91e1fe46f09b2c74f2dc4efe4f58ae59ae0a8/CHANGELOG).
+Marathon 1.6.392 [change log](https://github.com/mesosphere/marathon/releases).
+Metronome 0.4.2 [change log](https://github.com/dcos/metronome/releases/tag/v0.4.2).
+
+
+# <a name="issues-fixed"></a>Issues Fixed in DC/OS 1.11.2
```

Comment:
HTML in this heading is unnecessary (left over from previous bad practices). Pleas just use markdown: `# Issues Fixed in DC/OS 1.11.2`